src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,10 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import { UserInfo } from '../UserInfo';
+
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
```

Comment:
IMHO using classNames is better option.

Reply:
agreed about classnames, let's use it here